vscode-antimony/src/extension.ts

```diff
@@ -70,8 +76,42 @@ function updateDecorations() {
 	}
 }
 
+async function triggerSBMLEditor(event: TextDocument, sbmlFileNameToPath: Map<any, any>) {
+	if (path.extname(event.fileName) === '.xml') {
+		// check if the file is sbml, opens up a new file
+		await vscode.window.showTextDocument(event, { preview: true, preserveFocus: false });
+		  await vscode.commands.executeCommand('workbench.action.closeActiveEditor');
+		vscode.commands.executeCommand('antimony.sbmlFileToAntStr', event)
+			.then(async (result: any) => {

```

Comment:
indentation (as well as everything inside) .then is in line with vscode.commands.